shared/components/shopping/shopping.component.js

```diff
@@ -144,7 +144,8 @@ class ShoppingComponent extends Panel {
   }
 
   initializeServicesSlider(){
-    let shopping = this;
+    let shopping = this,
+        total_services_api_key = shopping.apiKey('input_footprint_shopping_services_total');
 
```

Comment:
Within the shopping component you overwrite the apiKey footprintable method checking if the type is part of the GOODS_QUESTIONS array.
So when using this method it returned `input_footprint_shopping_services_services_total` and `input_footprint_shopping_services_good_total`.

I have instead hard coded the keys for the total amounts within the services and goods slider init.